src/utils.php

```diff
@@ -189,3 +171,8 @@ function getVendorDir(\stdClass $composerJson = null): string
 {
     return $composerJson->config->{'vendor-dir'} ?? 'vendor';
 }
+
+function strStartsWith(string $haystack, string $prefix): bool
+{
+    return empty($prefix) || strpos($haystack, $prefix) === 0;
```

Comment:
While it is definitely more readable, function invocations have a considerable overhead in PHP compared to other languages, so I would avoid them if it's a one-liner.